benchmarks/go_benchmark_test.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"errors"
+	"io/ioutil"
+	"flag"
+	"testing"
+	"os"
+
+	benchmarkWrapper "./tmp"
+	proto "github.com/golang/protobuf/proto"
+	googleMessage1Proto3 "./tmp/datasets/google_message1/proto3"
+	googleMessage1Proto2 "./tmp/datasets/google_message1/proto2"
+	googleMessage2 "./tmp/datasets/google_message2"
+	googleMessage3 "./tmp/datasets/google_message3"
+	googleMessage4 "./tmp/datasets/google_message4"
+
+)
+
+// Data is returned by the Load function.
+type Data struct {
+	// Marshalled is a slice of marshalled protocol
+	// buffers. 1:1 with Unmarshalled.
+	Marshalled [][]byte
+
+	// Unmarshalled is a slice of unmarshalled protocol
+	// buffers. 1:1 with Marshalled.
+	Unmarshalled []proto.Message
+
+	count int
```

Comment:
Why is the other two using UpperCamelCase and this one using lower case?

Reply:
Done by changed the first two to lower case for not exporting outside package